contacts-app/src/components/ContactForm/ContactForm.jsx

```diff
@@ -14,65 +16,116 @@ class ContactForm extends React.Component {
         companyName: "",
         phone: "",
         email: ""
+      },
+      invalidMessage: {
+        firstName: "",
+        lastName: "",
+        phone: "",
+        email: ""
       }
     };
   }
 
   handleInputChange = event => {
     const update = {};
-    update[event.target.id] = event.target.value;
+    const inputId = event.target.id;
+    const inputValue = event.target.value;
```

Comment:
This can be done with descructuring too. 

```
const {id, value} = event.target;
```
or if you wanted to rename them like you have here you can add alias's
```
const {id: inputId, value: inputValue} = event.target;
```